### PR TITLE
update VK commitments to reflect scheduler fixes

### PR DESCRIPTION
**Pull request by akash-chandrakar**
## update VK commitments to reflect scheduler fixes
* update vk commitment
* update scheduler verification key

```
Running `target/debug/era-boojum-validator-cli --batch 175263 --network mainnet --l1-rpc 'my-rpc'
Fetching and validating the proof itself
Downloading proof for batch 175263 on network mainnet
Proof type: Scheduler
Will be evaluating Boolean constraint gate over specialized columns
Evaluating general purpose gates
Proof is VALID


Fetching data from Ethereum L1 for state roots, bootloader and default Account Abstraction parameters
Fetching batch 175262 information from zkSync Era on network mainnet
Fetching batch 175263 information from zkSync Era on network mainnet
Will be verifying a proof for state transition from root 0xa7caf9163e010a20c2d5e136dff69ca32d538f317a6fa62f13cde0a5fc4af536 to root 0x0e7bb8cc8264b7502617a06a9c47e88ae18d3c4035f811d428cb5c2780ca1f53
Will be using bootloader code hash 0x010007793c87889d295d21b454db454bc39fa21a1762714bcf643a95ce3594e6 and default AA code hash 0x0100067d1460ef6781360f14fe3ed8ddadb80d3be6ea33cb241afe5a07ab815b


Fetching auxilary block data
Downloading aux data for batch 175263 on network mainnet


Comparing public input from Ethereum with input for boojum 
Recomputed public input from current prover using L1 data is [0x0012a8e97dfb1c8c, 0x0055a22cf482087e, 0x006184f9455cb14a, 0x008482fb4091df11]
Boojum proof's public input is [0x0012a8e97dfb1c8c, 0x0055a22cf482087e, 0x006184f9455cb14a, 0x008482fb4091df11]
Boojum's proof is VALID
```

### CODE REVIEW

File: src/keys/verification_scheduler_key.json [hunk 1]
```diff
@@ -1,270 +1,270 @@
 {
```
Comment (mm-zk) [inline src/keys/verification_scheduler_key.json:1]: but changing this means, that the previous proofs will stop working, right?

Comment (akash-chandrakar) [reply src/keys/verification_scheduler_key.json:1]: updated.

File: src/main.rs [hunk 2]
```diff
@@ -51,12 +51,18 @@ pub fn proof_from_file<T: for<'a> Deserialize<'a>>(proof_path: &str) -> T {
     let proof: T = bincode::deserialize(buffer.as_slice()).unwrap();
     proof
 }
+fn get_scheduler_key_for_batch(batch_number: u64) -> &'static [u8] {
+    match batch_number {
+        1..=174710 => include_bytes!("keys/verification_scheduler_key.json"),
+        _ => include_bytes!("keys/verification_scheduler_key_174710.json"),
```
Comment (mm-zk) [inline src/main.rs:57]: could we name it 'v5'? (to be consistent with circuit naming?)

**Review by mm-zk — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (akashchandrakar0) [reply src/main.rs]: good point updated.
FYI: `node + leaf commitment update` might not result in `scheulder vk` update, so in scheduler we might have `v5`, `v9` etc, i.e not sequential order.

**Pull request merged**